packages/fxa-settings/src/lib/auth-errors/en.ftl

```diff
@@ -16,4 +16,5 @@ auth-error-138-2 = Unconfirmed session
 auth-error-139 = Secondary email must be different than your account email
 auth-error-155 = TOTP token not found
 auth-error-183-2 = Invalid or expired confirmation code
+auth-error-1003 = Local storage or cookies are still disabled
```

Comment:
tiniest of nits: this is "local storage or cookies" (local => cookies) &hellip; but below we refer to "cookies and local storage" (cookies => local) in 2 places.
Not sure if we want to reverse this to "Cookies or local storage are still disabled". Maybe not reader friendly considering i don't expect a bunch of end users to know what a "local storage" is anyways.

Reply:
Good point. I wasn't sure what was right between "is" or "are" and I'd asked my tech writer friend and she'd said this FWIW:

> It’s supposed to match the noun that’s closest to the verb, even if one is plural. It would be either “Local storage or cookies are still disabled” or “Cookies or local storage is still disabled.”

So, I'm guessing "Cookies and local storage are required" should technically be "Cookies and local storage is required". I'll check with Vesta that we want to go with the "Local storage" + "are" version for both.

Reply:
Oh, I don't know English. I was just commenting on order of "cookies" vs "local storage" in their relative strings.
But just a tiny nit, and I've wasted too much time on it already.

Reply:
That's what I meant I'd be double checking with Vesta about, sorry, that was just an FYI on "are" vs "is". Might as well before it gets localized 🤷‍♀️ 

She agrees we'll go with Local Storage first to match the error message.